apps/notifications/src/infrastructure/broker/config/index.ts

```diff
@@ -0,0 +1,10 @@
+import { registerAs } from '@nestjs/config';
```

Comment:
Small detail I noticed, we use double quotes `"` everywhere instead of single `'`

Reply:
The `.prettierrc.js` has it's own config instead of `module.exports = require("@akashnetwork/dev-config/.prettierrc");`

Reply:
thanks for spotting this!